VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -0,0 +1,143 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
+
+```
+ric@getmonero.org
+BDA6 BD70 42B7 21C4 67A9  759D 7455 C5E3 C0CD CEB9
+
+luigi1111@getmonero.org
+8777 AB8F 778E E894 87A2  F8E7 F4AC A018 3641 E010
+
+moneromooo.monero@gmail.com
+48B0 8161 FBDA DFE3 93AD  FC3E 686F 0745 4D6C EFC3
+```
+
+## II. Security Response Team
+
+- fluffypony
+- luigi1111
+- moneromooo
+
+## III. Incident Response
+
+1. Researcher submits report via one or both of two methods:
+    - a. Email
+    - b. [HackerOne](https://hackerone.com/monero)
+
+2. Response Team designates a Response Manager who is in charge of the particular report based on availability and/or knowledge-set
+
+3. In no more than 3 working days, Response Team should gratefully respond to researcher using only encrypted, secure channels
+
+4. Response Manager makes inquiries to satisfy any needed information to confirm if submission is indeed a vulnerability
+    - a. If submission proves to be vulnerable, proceed to next step
+    - b. If not vulnerable:
+      - i. Response Manager responds with reasons why submission is not a vulnerability
+      - ii. Response Manager moves discussion to a new or existing ticket on GitHub if necessary
+
+5. If over email, Response Manager opens a HackerOne issue for new submission
+
+6. Establish severity of vulnerability:
+    - a. HIGH: effects network as a whole, has potential to break entire network or is on a scale of great catastrophe
+    - b. MEDIUM: effects individual routers, or must be carefully exploited
+    - c. LOW: Is not easily exploitable
+
+7. Respond according to the severity of the vulnerability:
+    - a. HIGH severities must be notified on website and reddit /r/Monero within 3 working days of classification
+      - i. The notification should list appropriate steps for users to take, if any
+      - ii. The notification must not include any details that could suggest an exploitation path
+      - iii. The latter takes precedence over the former
+    - b. MEDIUM and HIGH severities will require a Point Release
+    - c. LOW severities will be addressed in the next Regular Release
+
+8. Response Team applies appropriate patch(es)
+    - a. Response Manager designates a PRIVATE git "hotfix branch" to work in
+    - b. Patches are reviewed with the researcher
+    - c. Any messages associated with PUBLIC commits during the time of review should not make reference to the security nature of the PRIVATE branch or its commits
+    - d. Vulnerability announcement is drafted
+      - i. Include severity of vulnerability
+      - ii. Include systems/apps effected
```

Comment:
If the language were "Include systems/apps which were affected by the vulnerability", that would also be accurate. Please see https://github.com/monero-project/monero/pull/1995#discussion_r112843832 as to why it's *effect*. 

Also, `to bring to pass` and `To effect` (the outcome of a produced affect):
```
Effect \Ef*fect"\, v. t. [imp. & p. p. {Effected}; p. pr. & vb.
   n. {Effecting}.]
   1. To produce, as a cause or agent; to cause to be.
      [1913 Webster]

            So great a body such exploits to effect. --Daniel.
      [1913 Webster]

   2. To bring to pass; to execute; to enforce; to achieve; to
      accomplish.
      [1913 Webster]

            To effect that which the divine counsels had
            decreed.                              --Bp. Hurd.
      [1913 Webster]

            They sailed away without effecting their purpose.
                                                  --Jowett (Th.
                                                  ).
```

So, either we rephrase/clarify, or trust the language 😉